setup.py

```diff
@@ -61,6 +61,7 @@ def run(self):
     "funcy>=1.12",
     "pathspec>=0.5.9",
     "shortuuid>=0.5.0",
+    "tqdm>=4.32.2",
```

Comment:
Just a note to ourselves: we'll need to update our conda package requirements for the next release.